rbac/management/access/view.py

```diff
@@ -19,12 +19,20 @@
 from management.cache import AccessCache
 from management.querysets import get_access_queryset
 from management.role.serializer import AccessSerializer
-from management.utils import APPLICATION_KEY, get_principal_from_request, validate_limit_and_offset
+from management.utils import (
+    APPLICATION_KEY,
+    get_principal_from_request,
+    validate_and_get_key,
+    validate_limit_and_offset,
+)
 from rest_framework.permissions import AllowAny
 from rest_framework.response import Response
 from rest_framework.settings import api_settings
 from rest_framework.views import APIView
 
+ORDER_FIELD = "order_by"
+VALID_ORDER_VALUES = ["unordered", "application", "resource_type", "verb", "-application", "-resource_type", "-verb"]
```

Comment:
Curious if `unordered` is necessary here or if it's superfluous since it's the default behavior when the param is not supplied?

Reply:
Yea, that util function to validate and get key is not allowing  None input. I am going to change that function 👍